rott/rt_cfg.c

```diff
@@ -1499,6 +1515,14 @@ void WriteSoundConfig
    SafeWriteString(file,"; 1 reverse stereo\n");
    WriteParameter (file,"StereoReverse      ",stereoreversed);
 
+   // write out soundfonts
```

Comment:
For the same reasons as above - i.e. soundfont files may come and go between compile time and run time, but also between two runs of the software - I don't think we should store the soundfont path in the config. Instead we should help the software to find a default soundfont at a fallback path.

Reply:
Sure, sounds good. I'll roll back these changes and figure out a better fallback.